Moblin/HaishinKit/Codec/VideoCodec.swift

```diff
@@ -133,6 +135,10 @@ class VideoCodec {
         ] status, _, imageBuffer, presentationTimeStamp, duration in
             guard let imageBuffer, status == noErr else {
                 logger.info("video: Failed to decode frame status \(status)")
+                guard let lastSampleBuffer else {
+                    return
+                }
+                delegate?.videoCodecOutputSampleBuffer(self, lastSampleBuffer)
```

Comment:
Is it really needed to output the last sample buffer? I mean, doesn't the sample buffer queue in the video unit already output the last buffer?

Reply:
That's right, with the current implementation this is no longer necessary. took it out